docs/types/conversion.rst

```diff
@@ -188,6 +188,11 @@ Addresses
 As described in :ref:`address_literals`, hex literals of the correct size that pass the checksum
 test are of ``address`` type. No other literals can be implicitly converted to the ``address`` type.
 
-Explicit conversions from ``bytes20`` or any integer type to ``address`` result in ``address payable``.
+Explicit conversions from ``bytes20`` or ``uint160`` to ``address`` is allowed and result in ``address payable``.
```

Comment:
This part is actually outdated. Starting with 0.8.0 you always need an explicit conversion to get a payable address.
```suggestion
Explicit conversions from ``bytes20`` or ``uint160`` to ``address`` are allowed.
```

Reply:
@cameel 
I have addressed your comment and removed this last part from the sentence (_"and result in ``address payable``."_
).

I have suggested to mention it in the next sentence then, to be more specific. See this commit: https://github.com/ethereum/solidity/pull/13105/commits/5f2be0c8b776d9eff750fd6bcb023a7e949e6c10

Let me know if that's ok, otherwise I will remove it.